prisma/schema.prisma

```diff
@@ -11,30 +11,31 @@ generator dbml {
 
 datasource db {
   provider = "postgresql"
-  url      = env("DATABASE_URL")
+  url      = "postgres://ssyvmyxygqknph:e84832604fe48e6469c7d10341ffff1fd43849752c534748fc2b5a8656e3d1b9@ec2-34-203-114-67.compute-1.amazonaws.com:5432/dfqebun1okmhlc"
```

Comment:
RIP DB, I'll generate a new connection string for you

Reply:
Okay, your new connection string is in the wiki

Reply:
Oh shoot, I thought I changed it. Ya, please do. I’ll fix the file. Thanks.